imports/ui/unit-explorer/filtered-units.jsx

```diff
@@ -41,8 +57,9 @@ export default class UnitsByRoles extends Component {
   }
 }
 
-UnitsByRoles.propTypes = {
+FilteredUnits.propTypes = {
   unitsByRoles: PropTypes.array,
+  searchResult: PropTypes.array,
```

Comment:
Why do you need both `searchResult` and `unitByRoles` if they do exactly the same?

Btw, `unitsByRoles` is not a good name, as it's just a flat array of units, it is not grouped to roles.

Reply:
`searchResults` and `unitsByRoles` are two different arrays of units filtered accordingly; 
`unitsByRoles` are grouped by roles via filters in filtered-units-container.jsx. `searchResults` are filtered by findUnit function.

This file(filtered-units.jsx) used to be units-by-roles.jsx. I changed to a more generic name, so that this component can be reused for `searchResults` in addition to `unitsByRoles`.

1) units-by-roles.jsx => filtered-units.jsx
2) filteredUnitsList.jsx => filtered-units-container.jsx